### PR TITLE
Open in new tab

### PR DESCRIPTION
**Pull request by gdpelican**
Ended up using `stopImmediatePropogation` to halt the navigation

Great success!

### CODE REVIEW

**Comment by robguthrie** [comment]
whoohoooo!

File: lineman/app/app.coffee [hunk 1]
```diff
@@ -22,9 +22,10 @@ angular.module('loomioApp', ['ngNewRouter',
     sanitize: true
     breaks: true
 
+  # TODO: I was secretly broken. Do I need to exist?
   # consume the csrf token from the page so form submissions can work
-  authToken = $("meta[name=\"csrf-token\"]").attr("content")
-  $httpProvider.defaults.headers.common["X-CSRF-TOKEN"] = authToken
+  # authToken = document.querySelector("meta[name=\"csrf-token\"]").attr("content")
```
Comment (robguthrie) [inline lineman/app/app.coffee]: FYI - I think we need to test on loomio-clone before deploying this one

File: lineman/app/components/lmo_href/lmo_href_for.coffee [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   restrict: 'A'
   scope:
     model: '=lmoHrefFor'
     action: '@lmoHrefAction'
   link: (scope, elem, attrs) ->
-    elem.attr 'href', ''
+    elem.attr 'href', LmoUrlService.route
+      model: scope.model
+      action: scope.action
     elem.bind 'click', ($event) ->
```
Comment (robguthrie) [inline lineman/app/components/lmo_href/lmo_href_for.coffee]: no angular new router dep right?

Comment (gdpelican) [inline lineman/app/components/lmo_href/lmo_href_for.coffee]: Nope.

Comment (gdpelican) [inline lineman/app/components/lmo_href/lmo_href_for.coffee]: The jquery ripout appears to have broken a bunch of the lineman specs, and
the e2ez don't really appear to be running either (only 3 assertions in 67
tests)

**Comment by robguthrie** [comment]
Ran out of time to tidy this up. Happy to do it tomorrow

**Pull request merged**